client/src/components/SelectDropdown/SelectDropdown.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./SelectDropdown.css";
+
+const SelectDropdown = props => (
+  <>
+    <select onClick={props.onClick}>
+      {props.option.map(element => (
+        <option value={element}>{element}</option>
+      ))}
+
+      {/* <option value={props.option}>{props.option}</option> */}
```

Comment:
This spacing and commented code should be removed if it is not needed.

Reply:
components/SelectDropdown/SelectDropdown.js
Shouldn't be included, i'm sorry.